tests/console/check_system_info.pm

```diff
@@ -81,6 +81,25 @@ sub check_buildid {
     }
 }
 
+# SLE-21783: Update nodejs-common for SLE15 SP4
+# check the nodejs16 is releaseed to SLE15 SP3 & SLE15 SP4
+# check the nodejs-common points to nodejs16
+# test steps:
+# 1) install nodejs-common on SLES15SP3 and migrated SLES15SP4 when wsm is available in SCC_ADDONS
+# 2) check nodejs16 is installed together with nodejs-common
+sub check_nodejs_common {
+    record_info('SLE-21783', 'check nodejs-common');
+    zypper_call('in nodejs-common | tee /tmp/install_nodejs.log');
+
+    assert_script_run('grep -E "Installing: nodejs16" /tmp/install_nodejs.log');
+    if ((script_output(q(zypper se -i nodejs | sed -n '/nodejs16 /p' | awk '{print $1}')) !~ /i/) && \
+        (script_output(q(zypper se -i nodejs | sed -n '/nodejs-common/p' | awk '{print $1}')) !~ /i\+/)) {
```

Comment:
&& -> || ?

Reply:
No you shouldn't change to ||.

Reply:
Oh, this need change back to ~=i && ~= i+
Or this logic is wrong

Reply:
updated the code using ||, thanks for review.